tests/validation/tests/v1_api/resource/deployment_1.json

```diff
@@ -0,0 +1,38 @@
+{
+  "type": "apps.deployment",
+  "metadata": {
+    "namespace": "",
+    "name": ""
+  },
+  "metadata.annotations": {},
+  "metadata.labels": {},
+  "spec": {
+    "replicas": 3,
+    "template": {
+      "spec": {
+        "restartPolicy": "Always",
+        "containers": [
+          {
+            "imagePullPolicy": "Always",
+            "volumeMounts": [],
+            "image": "sangeetha/mytestcontainer",
```

Comment:
Can we remove this placeholder and set it in the test? The image is a variable in the v3_api test code, and there's a high chance the repo for this image will change in the future.

Reply:
This is fixed by adding a new global variable:
```
TEST_IMAGE_V1 = os.environ.get('RANCHER_TEST_IMAGE_V1', "sangeetha/mytestcontainer")
```